src/org/traccar/protocol/EelinkProtocolDecoder.java

```diff
@@ -97,28 +114,72 @@ private String decodeAlarm(Short value) {
     }
 
     private void decodeStatus(Position position, int status) {
+
+        // GPS - 0 bit
```

Comment:
Here is an example of completely unnecessary comment:
```java
          // Charge - 7,8 bits
          if (BitUtil.check(status, 7)) {
              position.set(Position.KEY_CHARGE, BitUtil.check(status, 8));
          }
```

Actually, it's easier to understand the code than your comment.

We always save ACC as ignition value. We are not going to put the same comment in 100 places to explain it. You can put some comment in position table to clarify it, if you want.